docs/user_guide/reductions.rst

```diff
@@ -0,0 +1,429 @@
+The Reductions Approach to Disparity Mitigation
```

Comment:
There seems to be quite some overlap with content in mitigation.rst

Reply:
There is non-zero overlap, but it's not a large degree of overlap, given the difference in focus.

Reply:
Hmm, I still find it odd that there are multiple sections for this. With this it's basically not possible to read the theory (= content of this PR) along with the usage (code snippets) now. I had really hoped that we'd be able to achieve that because the paper is already a separate version of the theory that people can go and read.

Also, is this in any way related to the Overleaf doc? I'm wondering whether that's the same or similar in any way.

Reply:
@MiroDudik may have some thoughts on this. I think we need to settle on this before continuing or otherwise we risk wasting lots of time.

Reply:
I would expect things to be brought in from the Overleaf for sure, as something this gets filled out. Ideally I think that something like this would have collapsible sections, allowing users to dive deeper as they want. This of course means that the higher level sections should also read coherently without opening any of the lower levels.